vagrant/src/main/vagrant/servers.yaml

```diff
@@ -41,7 +41,9 @@ servers:
     box: ubuntu/wily64
     ram: 2048
     cpus: 4
-    ip: 10.10.10.100
+    forwarded_ports:
+     - guest: 8081
+       host: 8081
```

Comment:
If it's a process running in userspace then it will override the mapped port which will become accessible if the overriding process is halted.
Give me a second and I'll check the output from `:auto_correct`